flocker/cli/script.py

```diff
@@ -38,6 +43,29 @@
     b"images need to be pulled.\n")
 
 
+def treq_with_authentication(reactor, path):
+    """
+    Create a ``treq``-API object that implements the REST API TLS
+    authentication.
+
+    That is, validating the control service as well as presenting a
+    certificate to the control service for authentication.
+
+    :param reactor: The reactor to use.
+    :param FilePath path: Directory where certificates and private key can
```

Comment:
`certificate_path`?   `certificates`?  `path` is pretty generic, sort of like `number`.